src/tips/en/detect-unused-code.md

```diff
@@ -2,7 +2,7 @@
 date: 2023-03-03
 authors: Patrick Brosset
```

Comment:
Feel free to also add your name here. Similar to here: https://github.com/captainbrosset/devtools-tips/blob/b4e482b195559cdea774b56168ad97b1bb2ca359/src/tips/en/use-another-language.md?plain=1#L3

Reply:
Actually, never mind, I'll approve and merge, and then add this myself. Thanks for contributing Pankaj!